packages/eas-cli/src/graphql/queries/EnvironmentSecretsQuery.ts

```diff
@@ -27,18 +28,18 @@ export const EnvironmentSecretsQuery = {
       graphqlClient
         .query<EnvironmentSecretsByAppIdQuery>(
           gql`
-            query EnvironmentSecretsByAppId($appId: String!) {
+            query EnvironmentSecretsByAppId($appId: String!, $filterNames: [String!]) {
               app {
                 byId(appId: $appId) {
                   id
                   ownerAccount {
                     id
-                    environmentSecrets {
+                    environmentSecrets(filterNames: $filterNames) {
                       id
                       ...EnvironmentSecretFragment
                     }
                   }
-                  environmentSecrets {
+                  environmentSecrets(filterNames: $filterNames) {
```

Comment:
Why is this needed? It doesn't look like we use it anywhere.

Reply:
We keep environment secrets for compatibility, so you still have commands like `eas secret:create`.